product_video/models/media_video_provider.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MediaVideoProvider(models.Model):
+    _name = "media.video.provider"
+    _description = "Media Video Provider"
+
+    name = fields.Char()
+    code = fields.Char()
+    pattern_video_url = fields.Char(default="https://example.org/video/%s")
```

Comment:
all that 3 fields should be required, don't you think ?

Reply:
done